rofication/_gui.py

```diff
@@ -36,7 +36,8 @@ def rofi_entry(notification: Notification) -> str:
     stripped_summ = strip_tags(notification.summary)
     stripped_app = strip_tags(notification.application)
     stripped_body = strip_tags(' '.join(notification.body.split()))
-    return f'<b>{stripped_summ}</b> <small>({stripped_app})</small>\n<small>{stripped_body}</small>'
+    stripped_timestamp = f"{strip_tags (notification.timestamp)} " if notification.timestamp else ""
```

Comment:
I would like this UI addition to be opt-in and user-configurable.  It could be enabled via a commandline parameter or an Xresource key/value pair.

Reply:
Actually it is user configurable. That's what the parameter --tsformat does (on rotification daemon, which is called from regolith/i3/config). 

The default format with this PR is the system locale, but the right place to specify a default and document this parameter would be the config file itself, which for me looks like this:

```
  ###############################################################################
  # Rofication date & time format in Python format, see
  # https://docs.python.org/3/library/datetime.html
  # Defaults to locale default. Use empty string ("") for no timestamp
  ###############################################################################
  set_from_resource $rofication_datetime_format rofication_datetime_format "%x %X"
```
....
```
# Start Rofication for notifications
  set_from_resource $i3-wm.program.notifications i3-wm.program.notifications /usr/bin/rofication-daemon --tsformat $rofication  _datetime_format
  exec --no-startup-id $i3-wm.program.notifications...
```

Reply:
See this sibling PR:
https://github.com/regolith-linux/regolith-i3-gaps-config/pull/38/files

Reply:
Yes you're right, great.  

I think we can avoid a change in i3-gaps-config (the file you edited will not be shipped in Regolith 2.0 FYI) by directly reading from Xresources in rofication-daemon, and avoiding the command line arg as well.  This will localize the change entirely to rofication-dameon.  See [this utility class](https://github.com/regolith-linux/regolith-rofication/blob/master/rofication/_util.py#L20) for reading from env and xresources.

Reply:
FYI this is where i3 config lives in Regolith 2.0: https://github.com/regolith-linux/regolith-i3-config/tree/master/partials

Reply:
Changed.